docassemble/AssemblyLine/data/questions/ql_baseline.yml

```
@@ -2309,4 +2314,21 @@ content: |
 generic object: ALIndividual
 template: x.pronoun_self_described_label
 content: |
-  Self described pronouns
\ No newline at end of file
+  Self described pronouns
+---
+############# Contact method strings ################
+---
+generic object: ALIndividual
+template: x.phone_number_contact_label
+content: |
+  by phone at
```

I think we want:

```suggestion
content: >-
  by phone at
```

here, right? Haven't tested, but I always have to check https://yaml-multiline.info/ anway

---

Great catch, thanks!